services/workflowbuilder/src/main.ts

```diff
@@ -1,10 +1,21 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
   console.log(`⚡️ Running on http://localhost:3000`);
+
+  const config = new DocumentBuilder()
+    .setTitle('Cats example')
+    .setDescription('The WorkFlowBuilder API description')
+    .setVersion('1.0')
```

Comment:
Use the same version as the `services/workflowbuilder/package.json` so should be 0.0.1